File: MODUL 6/PRAK603-2310817210021-RAKHMAD FAUZHAN RAMADHAN.c
#include <stdio.h>

int main() {
    int n1, n2;
    scanf("%d %d", &n1, &n2);

    int baris1[n1], baris2[n2];
    
    for (int i = 0; i < n1; i++) {
        scanf("%d", &baris1[i]);
    }

    for (int i = 0; i < n2; i++) {
        scanf("%d", &baris2[i]);
    }

    if (n1 == n2) {
        for (int i = 0; i < n1; i++) {
            printf("%d ", baris1[i] * baris2[i]);
        }
        printf("\n");
    } else {
        printf("Jumlah tidak sama\n");
    }

    return 0;
}

File: MODUL 6/PRAK603-2310817210021-RAKHMAD FAUZHAN RAMADHAN.py
n1, n2 = map(int, input().split())

baris1 = list(map(int, input().split()))
baris2 = list(map(int, input().split()))

if n1 == n2:
    hasil = [baris1[i] * baris2[i] for i in range(n1)]

    print(*hasil)
else:
    print("Jumlah tidak sama")
